Makefile

```diff
@@ -28,7 +28,8 @@ endif
 GOARCH  ?= $(shell go env GOARCH)
 GOOS    ?= $(shell go env GOOS)
 
-VERSION     ?= $(shell git describe --tags --abbrev=7)
+OS_GIT_VERSION ?= unknown
+VERSION     ?= $(OS_GIT_VERSION)
```

Comment:
Is it worth having this fall back to the git approach if that env var doesn't exist?

Reply:
could switch the logic here to use the old method when not found. i don't think it makes a huge difference either way since this value is used when we make the images for release, but i don't think it would hurt either.

Reply:
updated the logic here